src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +136,27 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testAesdefaultmodepadding() throws InvalidParametersException {
+        String msg = "Test AES Default";
+        String decrptedMsg;
+        byte[] decrypted;
+        decrptedMsg = StringHelper.encrypt(msg);
+
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.DECRYPT_MODE, sks);
+            decrypted = cipher.doFinal(StringHelper.hexStringToByteArray(decrptedMsg));
```

Comment:
Actually i have also tried this way earlier by extracting default  mode and padding but was unable to find so shifted to this way.

So now we are testing by encrypting by "AES/ECB/PKCS5Padding" and decrypt it by default "AES" ?